.vscode/extensions.json

```diff
@@ -0,0 +1,14 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
```

Comment:
broken json

Reply:
huh. i think it still works but i'll remove it anyway

Reply:
It's jsonc actually. Just like `tsconfig.json`.